plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -298,8 +298,8 @@ public void testOptimizingPartitionsOfV2TableWithGlobalEqualityDeleteFile()
         List<String> updatedFiles = getActiveFiles(tableName);
         Assertions.assertThat(updatedFiles)
                 .doesNotContain(initialActiveFiles.stream()
-                                .filter(path -> !path.contains("regionkey=1"))
-                                .toArray(String[]::new));
+                        .filter(path -> !path.contains("regionkey=1"))
+                        .toArray(String[]::new));
```

Comment:
is this change needed? If yes then it should be in a separate commit.

Reply:
Oh...this might be related with my code format. It's not needed...let me revert this in the new commit.